opm/parser/eclipse/EclipseState/SimulationConfig/tests/ThresholdPressureTest.cpp

```diff
@@ -181,13 +181,10 @@ static DeckPtr createDeck(const ParseContext& parseContext , const std::string&
     return parser.parseString(input , parseContext);
 }
 
-static Eclipse3DProperties getEclipseProperties(const Deck& deck)
-{
-    static const EclipseGrid eclipseGrid( 3, 3, 3 );
-    return Eclipse3DProperties( deck, TableManager( deck ), eclipseGrid );
+static Eclipse3DProperties getEclipseProperties(const Deck& deck) {
+    return Eclipse3DProperties(deck, *new TableManager(deck), *new EclipseGrid(3, 3, 3));
```

Comment:
The other test code could really use some love if they're to be good for educational purposes.

Patches welcome.